src/unitxt/llm_as_judge.py

```diff
@@ -71,7 +71,11 @@ def _get_instance_for_judge_model(
                 {
                     "question": input_instance,
                     "answer": prediction,
-                    "reference_answer": reference,
+                    "reference_answer": str(
```

Comment:
I think the metric should take the first reference - and also set the metric 

single_reference_per_prediction: bool = True